### PR TITLE
Feat: Added rules page.

### PR DESCRIPTION
**Pull request by coldpigli**
Added the rules page

image

### CODE REVIEW

File: styles/rules.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.rules-cta{
+    gap: var(--space-md);
+}
+
+.rules-cta>button{
+    width: 15rem;
+    border: none;
+    border-radius: 0%;
+    background-color: var(--pastel-red);
```
Comment (shuntzhere) [inline styles/rules.css:21]: You can just remove border-radius.

**Review by shuntzhere — COMMENTED** [review]
HTML is very clean and easy to read. Suggested a minor change rest looks pretty good to me.

File: styles/rules.css (hunk 1 above)
Comment (coldpigli) [reply styles/rules.css:21]: Hi, the border radius is defined here is to override the default value that comes from the component library I have used. So it's serving a purpose here.

**Review by sadathshariff — APPROVED** [review]
Everything looks clean and good

File: styles/rules.css (hunk 1 above)
Comment (shuntzhere) [reply styles/rules.css:21]: Ah, that makes sense.

**Review by shuntzhere — COMMENTED** [review]
Everything looks clean and good to go.

**Pull request merged**